pydantic/_internal/_generics.py

```diff
@@ -184,7 +184,7 @@ def _get_caller_frame_info(depth: int = 2) -> tuple[str | None, bool]:
 DictValues: type[Any] = {}.values().__class__
 
 
-def iter_contained_typevars(v: Any) -> Iterator[TypeVarType]:
+def iter_contained_typevars(v: Any) -> Iterator[TypeVar]:
```

Comment:
Could you explain this change, briefly? Thanks!

Reply:
`TypeVarType` was defined as an alias to `Any` as a workaround, because mypy could not understand something like this:

```python
TypeVarType: TypeAlias = TypeVar
```

But for pyright (and maybe mypy as of today), it is perfectly fine to use `TypeVar` as type annotations, it behaves as any other type